core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -659,6 +680,24 @@ default Iterable<Entry<String,String>> getProperties(String tableName)
   Map<String,String> getConfiguration(String tableName)
       throws AccumuloException, TableNotFoundException;
 
+  /**
+   * Gets per-table properties of a table. This operation is asynchronous and eventually consistent.
+   * It is not guaranteed that all tablets in a table will return the same values. Within a few
+   * seconds without another change, all tablets in a table should be consistent. The clone table
+   * feature can be used if consistency is required. This new method returns a Map instead of an
```

Comment:
Do we need to say "This new method returns a Map instead of an Iterable."? If we leave this in place (in multiple places), maybe we should add something to say why?

Reply:
I don't think so, I am just going to drop that clause.